apiary.apib

```diff
@@ -3315,6 +3317,151 @@ Get information about a meeting.
                     + open
                     + done
 
+## Calls [/calls]
+
+### calls.add [POST /calls.add]
+
+Add a new call.
+
++ Request (application/json)
+
+    + Attributes
+        + description: `Description of the call` (string, optional)
+        + customer (object, required)
+            + type (enum, required)
+                + Members
+                    + company
+                    + contact
+            + id: `2659dc4d-444b-4ced-b51c-b87591f604d7` (string, required)
+        + due_date: `2016-02-04T16:00:00+00:00` (string, required)
```

Comment:
We only talk about `date` when talking about a day without referencing the time, since we are also talking about a time, this thus must be `due_on`

```suggestion
        + due_on: `2016-02-04T16:00:00+00:00` (string, required)
```

On the other hand, we should pull this in one line with the response property `scheduled_on`. Given that I think we can go with `due_on` instead of `scheduled_on` or `_for` I'm not sure which one is the most correct term to be used in this case.